xbmc/pvr/PVRPlaybackState.h

```diff
@@ -76,7 +76,8 @@ class CPVRPlaybackState
    */
   void StartPlayback(
       CFileItem* item,
-      ContentUtils::PlayMode mode = ContentUtils::PlayMode::CHECK_AUTO_PLAY_NEXT_ITEM) const;
+      ContentUtils::PlayMode mode = ContentUtils::PlayMode::CHECK_AUTO_PLAY_NEXT_ITEM,
+      bool fromEpgAsLive = false) const;
```

Comment:
Do you see a way to avoid the bool param, for example by providing an new method? I'm not a fan of bool parameters, because resulting calling code is (almost ever) not really readable.

And please, no default parameters. Those are for the lazy guys only. :-)

Reply:
Enum again I guess.

Reply:
`enum class` would do the trick, yes.

Reply:
I'll see if I can remove all the optional params from the StartPlayback call. Let me do some digging.